locksmith/src/controllers/v2/checkoutController.ts

```diff
@@ -34,13 +34,20 @@ export const createOrUpdateCheckoutConfig: RequestHandler = async (
       createdBy: request.user!.walletAddress,
     })
 
+    if (!createdConfig) {
+      return response.status(403).send({
+        message:
+          'Unauthorized: You do not have permission to create or update this configuration.',
+      })
+    }
+
     return response.status(200).send({
-      id: createdConfig.id,
-      by: createdConfig.createdBy,
-      name: createdConfig.name,
-      config: createdConfig.config,
-      updatedAt: createdConfig.updatedAt.toISOString(),
-      createdAt: createdConfig.createdAt.toISOString(),
+      id: createdConfig?.id,
+      by: createdConfig?.createdBy,
+      name: createdConfig?.name,
+      config: createdConfig?.config,
+      updatedAt: createdConfig?.updatedAt.toISOString(),
+      createdAt: createdConfig?.createdAt.toISOString(),
     })
```

Comment:
I don't think you need to add the `?` here because if `createdConfig` is not defined, the above will catch it?